packages/cli/src/commands/translate/sync-keys/utils.ts

```diff
@@ -96,13 +99,13 @@ export const praseLocaleFiles = (filePath: string): ParsedTuple => {
         // Recursively parse the nested object from the imported file
         const [phrases, structure] = praseLocaleFiles(resolvedPath);
 
-        // eslint-disable-next-line @silverhand/fp/no-mutation
+        /* eslint-disable @silverhand/fp/no-mutation */
```

Comment:
Is this expected? Seems all `//` are replaced by `/* */`

Reply:
yes, save some lines